### ISSUE TITLE
Only posts from one user in an issue

### ISSUE THREAD

**Issue by meneal** [CSC510-2015-Axitron/project2#7]
It seems like a bad thing if only one person ever posts on a particular issue.  It really means that there is a certain level of disengagement from other team members.

**Comment by duh102** [comment]
You can get the number of (unique) users commenting on an issue with

``` SQL
select issueID, count(distinct user) from comment group by issueID;
```

or, if you just want the issues that have only one person commenting on them,

``` SQL
select issueID from (select issueID, count(distinct user) as uUsers from comment group by issueID) where uUsers == 1;
```

**Comment by duh102** [comment]
So this one is a bit difficult to graph, because the data is so discretized, but I tried a boxplot.
![boxplot_users_in_issues](https://cloud.githubusercontent.com/assets/6590396/7281781/b7dc18da-e8f9-11e4-838a-3d9010f708ce.png)

**Comment by meneal** [comment]
What's the deal with groups 2,5, and 7?  It looks from this like the median
Q1, and Q3 were all the same value, 1.  Then even having 2 users writing on
an issue was an outlier.  That seems like a bad smell, don't you think?  I
agree with you though that this is a bit of a tough one to graph.  I'm not
sure what would be better, but if how I'm interpreting this is correct then
it's pretty easy to use for what we need.

**Comment by duh102** [comment]
Yeah, those groups had large amounts of people only commenting on one issue (group 7 only had one person in it), group 5 had some more issues with comments from two people, but the mean was still 1 person.
